tools/topology/m4/detect_test_coef.m4

```diff
@@ -6,6 +6,6 @@ CONTROLBYTES_PRIV(DETECTOR_priv,
 `       0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,'
 `       0x20,0x00,0x00,0x00,0x00,0x00,0x00,0x00,'
 `       0x00,0x00,0x00,0x00,0x34,0x08,0x00,0x00,'
-`       0x03,0x00,0xff,0x3f,0x34,0x08,0x00,0x00,'
-`       0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00"'
+`       0x03,0x00,0x18,0x00,0x00,0x00,0x00,0x00,'
```

Comment:
are you setting activation_threshold to be 0 here?

Reply:
yes its zero and FW uses default values.